trunk/projects/h5fs_test/test_data_to_hdf5

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
I would rather see this kind of test script saved in jenkins than into the repo. I'm not sure this belongs in the repo